resources/dicts/events/death/beach/apprentice.json

```diff
@@ -1,3 +1,225 @@
+        "death_text": "m_c knew {PRONOUN/m_c/subject} made a mistake, as {PRONOUN/m_c/subject} felt {PRONOUN/m_c/poss} last bits of strength slip away from {PRONOUN/m_c/object} and {PRONOUN/m_c/poss} heart stopped beating. {PRONOUN/m_c/poss/CAP} body only discovered days later, r_c wailing at the shriveled body of {PRONOUN/r_c/poss} apprentice, who succumbed to heat and dehydration, covered in beetles.",
+        "history_text": {
+                        "reg_death": "m_c succumbed to dehydration and heat exhaustion."
+                        },
+        "cat_trait": null,
+        "cat_skill": null,
+        "other_cat_trait": null,        
+        "other_cat_skill": null  
+     },
+     {
+        "camp": "any",
+        "tags": [
+         "Newleaf", 
+         "Greenleaf", 
+         "Leaf-fall",
+         "other_cat",
+         "other_cat_adult"
+        ],
+        "death_text": "It was purely accidental that r_c stumbled across m_c's decaying body in the sunlight, simply hoping to stretch {PRONOUN/r_c/poss} paws outside of camp. Now, r_c must explain to m_c's remaining family or friends that {PRONOUN/m_c/subject} {VERB/m_c/are/is} never returning home, having been caught unprepared under the treacherous sun.",
+        "history_text": {
+                     "reg_death": "m_c was caught outside unprepared for the heat, dying because of dehydration and overheating."
+                 },
+        "cat_trait": null,
+         "cat_skill": null,
+         "other_cat_trait": null,        
+         "other_cat_skill": null 
+     },
+     {
+        "camp": "any",
+        "tags": [
+         "Newleaf", 
+         "Greenleaf", 
+         "Leaf-fall",
+         "other_cat",
+         "other_cat_parent"
+        ],
+        "death_text": "m_c  never meant to meet StarClan this way, rotting away under the sun after fainting from dehydration, heat exhaustion, and overexertion. r_c tragically discovers {PRONOUN/m_c/poss} body, unable to stop {PRONOUN/r_c/poss} desperate wails at the site of {PRONOUN/r_c/poss} child's decaying corpse, being feasted on by butterflies and other scavengers.",
+        "history_text": {
+                     "reg_death": "m_c tragically died due to dehydration and overheating."
+                 },
+        "cat_trait": null,
+         "cat_skill": null,
+         "other_cat_trait": null,        
+         "other_cat_skill": null 
+     },
+     {
+        "camp": "any",
+        "tags": [
+         "Newleaf", 
+         "Greenleaf", 
+         "Leaf-fall",
+         "other_cat",
+         "other_cat_adult"
+        ],
+        "death_text": "r_c's mournful sounds pierced through c_n's territory as {PRONOUN/r_c/subject} stumble on the lifeless, decaying corpse of m_c. m_chad disappeared from camp days ago, having not returned, but only now did r_c understand why  {PRONOUN/r_c/poss} child didn't come back home; Rose was caught out in the treacherous sun, ill-prepared, succumbing to dehydration and exhaustion.",
```

Comment:
Typo: There is missing a space between "m_c" and "has" at the beginning of the second sentence.